src/Logging/src/SerilogDynamicLoggerCore/SerilogDynamicProvider.cs

```diff
@@ -44,279 +35,89 @@ public SerilogDynamicProvider(IConfiguration configuration, Logger logger, Loggi
         /// <param name="logger">Serilog logger<see cref="Serilog.Core.Logger"/></param>
         /// <param name="loggingLevelSwitch">Serilog global log level switch<see cref="Serilog.Core.LoggingLevelSwitch"/></param>
         /// <param name="options">Subset of Serilog options managed by wrapper<see cref="ISerilogOptions"/></param>
-        public SerilogDynamicProvider(IConfiguration configuration, ISerilogOptions options = null, Logger logger = null, LoggingLevelSwitch loggingLevelSwitch = null)
+        [Obsolete("Will be removed in a future release; Use SerilogDynamicProvider(IOptionsMonitor<SerilogOptions>, IEnumerable<IDynamicMessageProcessor>) instead ")]
+        public SerilogDynamicProvider(IConfiguration configuration, ISerilogOptions options, Logger logger = null, LoggingLevelSwitch loggingLevelSwitch = null)
```

Comment:
Removing null default for `ISerilogOptions options` is a breaking change, is it required?

Reply:
Yes its required and I don't believe its breaking. Not doing this give this error: 

```
  System.InvalidOperationException : Unable to activate type 'Steeltoe.Extensions.Logging.SerilogDynamicLogger.SerilogDynamicProvider'. The following constructors are ambiguous:
    Void .ctor(Microsoft.Extensions.Configuration.IConfiguration, Steeltoe.Extensions.Logging.SerilogDynamicLogger.ISerilogOptions, Serilog.Core.Logger, Serilog.Core.LoggingLevelSwitch)
    Void .ctor(Microsoft.Extensions.Options.IOptionsMonitor`1[Steeltoe.Extensions.Logging.SerilogDynamicLogger.SerilogOptions], System.Collections.Generic.IEnumerable`1[Steeltoe.Extensions.Logging.IDynamicMessageProcessor])
```